src/Modules/PaymentModule/components/PaymentGetaway/PaymentGetaway.tsx

```diff
@@ -0,0 +1,18 @@
+import { Elements } from "@stripe/react-stripe-js";
+import { loadStripe } from "@stripe/stripe-js"
+import { CheckoutForm } from "../CheckoutForm/CheckoutForm";
+export const PaymentGetaway = () => {
+
+
+  const stripe = loadStripe(
```

Comment:
This should be outside of the components, to prevent loading the stripe elements when the components re-renders. Having it outside of the components mean that it only loads in the start of our application.